cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModelExt.kt

```diff
@@ -121,7 +124,8 @@ internal fun CCPA.toCcpaCS() = CcpaCS(
     ccpaApplies = null,
     uuid = null,
     gpcEnabled = null,
-    actions = null
+    actions = null,
```

Comment:
@bohdan-go-wombat this bad boy `actions` might be giving me a headache in another ticket [DIA-2273](https://sourcepoint.atlassian.net/browse/DIA-2273)
I'm interested in your opinion: is it safe to obliterate it from the model or it is crucial to the SDK architecture?

Reply:
judging by a quick investigation, it doesn't seem like it is needed at all in our sdk